### PR TITLE
LIVE-3703 Disable countervalue for Cardano tokens unsupported in CoinGecko

### PR DESCRIPTION
**Pull request by haammar-ledger**
### 📝 Description

It's all in the title.

### ❓ Context

- **Impacted projects**: `@ledgerhq/cryptoassets` 
- **Linked resource(s)**: https://ledgerhq.atlassian.net/browse/LIVE-3703 

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

#### Accounts screen
image

#### Token account screen
image

### CODE REVIEW

File: .github/workflows/bot-cardano-silicium.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ on:
   push:
     branches:
       - feat/cardano
+      - feat/LIVE-3703-ada-disable-countervalues
```
Comment (henrily-ledger) [inline .github/workflows/bot-cardano-silicium.yml:6]: minor but it can be removed ?

**Review by henrily-ledger — APPROVED** [review]

**Comment by henrily-ledger** [comment]
Error on bot need to reregen

File: .github/workflows/bot-cardano-silicium.yml (hunk 1 above)
Comment (hedi-edelbloute) [inline .github/workflows/bot-cardano-silicium.yml:6]: Might need to remove this before merging

**Review by hedi-edelbloute — APPROVED** [review]

**Review by gre — APPROVED** [review]

**Comment by gre** [comment]
did you run the integration tests too?

**Comment by haammar-ledger** [comment]
> did you run the integration tests too?

Good point :+1: run started here:
https://github.com/LedgerHQ/ledger-live/actions/runs/3249025560

**Comment by haammar-ledger** [comment]
@gre these errors seem to exist on develop as well... :cry: (at least I don't think I see new ones?)
https://github.com/LedgerHQ/ledger-live/actions/runs/3248888825/jobs/5330640427

That's the main issue with integration tests, we never managed to stabilize them yet.

**Comment by haammar-ledger** [comment]
Errors on integ tests aren't related to this PR.
Validated by QA, merging to avoid blocking this work, tests will be addressed separately.

**Pull request merged**